packages/wonder-blocks-core/util/server.test.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import Server from "./server.js";
+
+describe("./server.js", () => {
+    it("#isServerSide should return false", () => {
+        // Arrange
+
+        // Act
+        const result = Server.isServerSide();
+
+        // Assert
+        expect(result).toBeFalsy();
+    });
+
+    it("#setServerSide should set server-side mode to true", () => {
+        // Arrange
+
+        // Act
+        Server.setServerSide();
+        const result = Server.isServerSide();
+
+        // Assert
+        expect(result).toBeTruthy();
+    });
+
+    it("#setServerSide should leave server-side mode as true", () => {
+        // Arrange
+
+        // Act
+        Server.setServerSide();
+        const result = Server.isServerSide();
+
+        // Assert
+        expect(result).toBeTruthy();
+    });
```

Comment:
This test is the same as the previous test.  Is it necessary?

Reply:
It's missing a call to `Server.setServerSide()` so that it is verifying nothing changes, since that's a behavior we expect, I want a test to make sure that doesn't change.